.azure/pipelines/richnav.yml

```diff
@@ -6,58 +6,68 @@
 trigger:
   branches:
     include:
-    - blazor-wasm
     - main
     - release/*
-    - internal/release/*
+
+# Do not run this pipeline for PR validation.
+pr: none
 
 variables:
 - name: _BuildArgs
   value: '/p:SkipTestBuild=true'
-- name: Windows86LogArgs
+- name: WindowsNonX64LogArgs
   value: -ExcludeCIBinaryLog
 
 stages:
 - stage: build
   displayName: Build
   jobs:
-  # Build Windows (x64/x86)
+  # Build Windows (x64/x86/arm64)
   - template: jobs/default-build.yml
     parameters:
       jobName: Windows_build
-      jobDisplayName: "Build: Windows x64/x86"
+      jobDisplayName: "Build: Windows x64/x86/arm64"
       enableRichCodeNavigation: true
       agentOs: Windows
       steps:
-      - script: ./build.cmd
+      - script: ./eng/build.cmd
                 -ci
-                -all
                 -arch x64
-                /p:EnableRichCodeNavigation=true
+                -buildNative
+                /p:EnableRichCodeNavigation=false
```

Comment:
Setting things this way overrides the environment variable created for the `$(enableRichCodeNavigation)` variable job.yml sets based on the parameter. Given https://dev.azure.com/dnceng/public/_build/results?buildId=1667542&view=results worked, this seems about right though I wonder how repos are successfully processing C++ code. Perhaps they all use `cmake` instead of `cl`❔